api/workerapi/workerrpcclient/client.go

```diff
@@ -0,0 +1,192 @@
+package workerrpcclient
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	v1 "github.com/iver-wharf/wharf-cmd/api/workerapi/v1"
+	"github.com/iver-wharf/wharf-core/pkg/logger"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+// Client is used to communicate with the Worker gRPC server.
+type Client struct {
+	targetAddress string
+	targetPort    string
+
+	Client v1.WorkerClient
+	conn   *grpc.ClientConn
+}
+
+var log = logger.NewScoped("WORKER-RPC-CLIENT")
+
+// NewClient creates a new gRPC Client that can communicate with the Worker
+// gRPC server.
+func NewClient(targetAddress, targetPort string) *Client {
+	return &Client{
+		targetAddress: targetAddress,
+		targetPort:    targetPort,
+	}
+}
```

Comment:
Suggest just merging the targetAddress and targetPort. I see no need to keep them separate.

It also stays conventional with the rest of wharf-api, wharf-providers, and wharf-cmd-provisioner, where we only pass in a single "bindAddress".

Reply:
Merged 👍🏻
Used the name `address` for the clients as they are not binding.